docs/dev-tools/zombienet/overview.md

```diff
@@ -235,417 +243,319 @@ Zombienet provides a CLI that allows interaction with the tool. The CLI can rece
-|  `test`   |                                  Run test on the network spawned                                   |                                                                      `<testFile>` - a file that defines assertions and tests against the spawned network, using natural language expressions to evaluate metrics, logs, and built-in functions                                                                       |
-|  `setup`  |                           Set up the dev environment of Zombienet ready.                           |                                                                                          `<binaries>` - executables that will be downloaded and prepared to be used by Zombienet. Options: `polkadot`, `polkadot-parachain`                                                                                          |
-| `convert` | Transforms a (now deprecated) polkadot-launch configuration file to a zombienet configuration file | `<filePath>` - path to a [Polkadot Launch](https://github.com/paritytech/polkadot-launch){target=_blank} configuration file with a .js or .json extension defined by [this structure](https://github.com/paritytech/polkadot-launch/blob/295a6870dd363b0b0108e745887f51e7141d7b5f/src/types.d.ts#L10){target=_blank} |
-| `version` |                                      Prints Zombienet version                                      |                                                                                                                                                          -                                                                                                                                                           |
-|  `help`   |                                      Prints help information                                       |                                                                                                                                                          -                                                                                                                                                           |
+### CLI Commands
 
+??? function "`spawn` - spawn the network defined in the config file"
 
-!!! warning
+    === "Argument"
+
+        - `<networkConfig>` - a file that declares the desired network to be spawned in `.toml` or `.json` format. For further information, check out the [Configuration Files](#configuration-files) section
+
+    !!! warning
     For the `spawn` command to work on macOS, users need to be aware that the Polkadot binary is currently not compatible with macOS. As a result, macOS users will need to clone the [Polkadot repository](https://github.com/paritytech/polkadot-sdk){target=_blank}, build Polkadot binary, and manually add it to their PATH.
```

Comment:
This warning isn't rendering properly because the content inside the warning needs to be indented 4 more spaces

For example, this does not work:

```
!!! note
Doesn't render properly.
```

But this does work:

```
!!! note
    Now it will render as this text is indented 4 spaces.
 ```